core/src/test/java/com/klaytn/caver/legacy/feature/CustomGasTest.java

```diff
@@ -67,32 +70,32 @@ public void testCustomGasPriceWithValueTransfer() throws Exception {
                 sender.getAddress(),
                 to.getAddress(),
                 value,
-                GAS_PRICE,
+                gasProvider.getGasPrice(),
                 GAS_LIMIT
         );
-        assertEquals(tx.getGasPrice(), GAS_PRICE);
+        assertEquals(tx.getGasPrice(), gasProvider.getGasPrice());
```

Comment:
Before `gasProvider.getGasPrice` might be different with after processing the tx `gasProvider.getGasPrice`.
How about making a variable to set `suggestedGas = gasProvider.getGasPrice()` and compare with that?